documentation/release_notes.rst

```diff
@@ -8,6 +8,51 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2022.9.0
+===============
+
+New Features
+------------
+- Added parallel range algorithms in ``namespace oneapi::dpl::ranges``: ``fill``, ``move``, ``replace``, ``replace_if``,
+``remove``, ``remove_if``, ``mismatch``, ``minmax_element``, ``min``, ``max``, ``find_first_of``, ``find_end``, ``is_sorted_until``
+- Improved performance of set operation algorithms (e.g., ``set_union``, ``set_difference``, etc.)
+
+Fixed Issues
+------------
+- Eliminated runtime exceptions that were encountered when compiling code that uses device policies with the open source
+oneAPI DPC++ Compiler (clang++ driver) without specifying an optimization flag.
+- Eliminated compile time warnings throughout the library.
+
+
+Known Issues and Limitations
+----------------------------
+New in This Release
+^^^^^^^^^^^^^^^^^^^
+
+Existing Issues
+^^^^^^^^^^^^^^^
+See oneDPL Guide for other `restrictions and known limitations`_.
+
+- Incorrect results may be observed when calling ``sort`` with a device policy on Intel® Arc™ graphics 140V with data
+  sizes of 4-8 million elements.
```

Comment:
```suggestion
- ``sort``, ``stable_sort``, ``sort_by_key`` and ``stable_sort_by_key`` algorithms with a device policy are not supported
  on Windows with Intel® Arc™ graphics 140V yet.
```

Let's better say that it is not supported yet - the HW is relatively new.

Reply:
Ah, you are right. Let's keep as is here. I hope to investigate and fix it by the next release.